container/karaf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -105,6 +105,8 @@ featuresBoot = ( \
   opennms-telemetry-jti,\
   opennms-telemetry-nxos,\
   opennms-telemetry-graphite,\
+  opennms-telemetry-openconfig-client,\
```

Comment:
Wouldn't we expect `opennms-telemetry-openconfig` to also load `opennms-telemetry-openconfig-client`?